src/bin/pgcopydb/env_utils.h

```diff
@@ -8,6 +8,29 @@
 
 #include "postgres_fe.h"
 
+enum EnvType
+{
+	ENV_TYPE_STRING,
+	ENV_TYPE_INT,
+	ENV_TYPE_BOOL
+};
+
+typedef struct EnvParser
+{
+	char *envname;      /* name of the environment variable */
+	enum EnvType type;
+	void *target;
+	int targetSize;     /* for strings, 0 for other types */
+	int minValue;       /* for integers, 0 if there is no lower bound */
```

Comment:
We might have a use for zero as the lower bound, I would use -1 here. What do you think?

Reply:
I used to be able to leave out the last members in struct assignments as I knew they will be filled with zeros. Now that the default is -1, I explicitly set the values of all the members.

Although the earlier method looked better to me, I can see the benefit of showing it explicitly that the allowed values of the options are not bounded.

Reply:
You could also introduce a boolean such as "bounded", default is false and then no magic values are needed anymore.

Reply:
I introduced `lowerBounded` and `upperBounded` struct members.